src/constants/room_ids.asm

```diff
@@ -0,0 +1,4 @@
+; values for hMapRoom
+
+OVERWORLD_KANALET_CASTLE_CROW_ROOM      equ $58
+OVERWORLD_KANALET_CASTLE_FIVE_PITS_ROOM equ $5A
```

Comment:
Excellent ; there are more constant-ed rooms in the code that will also benefit from having a list of room constants.